### ISSUE TITLE
Warning message with the AsnicarF_2017.relative_abundance data set when counts are set to TRUE in the cMD function.

### ISSUE THREAD

**Issue by sdgamboa** [waldronlab/curatedMetagenomicData#206]
Warning message with the AsnicarF_2017.relative_abundance data set when counts are set to TRUE in the `curatedMetagenomicData` function.

>`Warning in magrittr::multiply_by(., colData[["number_reads"]]): longer object`
>`length is not a multiple of shorter object length`

``` r
library(curatedMetagenomicData)
curatedMetagenomicData("AsnicarF_2017.relative_abundance", dryrun = FALSE, counts = TRUE)
#> snapshotDate(): 2021-04-08
#> see ?curatedMetagenomicData and browseVignettes('curatedMetagenomicData') for documentation
#> loading from cache
#> Warning in magrittr::multiply_by(., colData[["number_reads"]]): longer object
#> length is not a multiple of shorter object length
#> class: SummarizedExperiment 
#> dim: 550 24 
#> metadata(0):
#> assays(1): relative_abundance
#> rownames(550): k__Bacteria k__Bacteria|p__Proteobacteria ...
#>   k__Bacteria|p__Firmicutes|c__Bacilli|o__Lactobacillales|f__Streptococcaceae|g__Streptococcus|s__Streptococcus_gordonii
#>   k__Bacteria|p__Firmicutes|c__Bacilli|o__Lactobacillales|f__Aerococcaceae|g__Abiotrophia|s__Abiotrophia_sp_HMSC24B09
#> rowData names(0):
#> colnames(24): MV_FEI1_t1Q14 MV_FEI2_t1Q14 ... MV_MIM5_t2M14
#>   MV_MIM5_t3F15
#> colData names(20): studyName subjectID ... curator NCBI_accession
```

<sup>Created on 2021-04-20 by the [reprex package](https://reprex.tidyverse.org) (v2.0.0)</sup>

**Comment by schifferl** [comment]
Closed with #188.

**Comment by lwaldron** [comment]
I think this issue is unrelated to #188 or its Trello card, because this has to do with matrix multiplication being performed by the `counts = TRUE` function. I would not close this issue until there is a unit test confirming that conversion to counts is being done correctly.

**Comment by schifferl** [comment]
This is resolved by ceb7958 and the issue can be closed now, great find @sdgamboa (and persistence @lwaldron). If you're curious, the matrix is subset (now anyway) along with the meta data to avoid any size differences.
